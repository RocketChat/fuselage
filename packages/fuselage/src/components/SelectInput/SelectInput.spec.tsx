import React from 'react';
import ReactDOM from 'react-dom';

import { SelectInput } from '.';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<SelectInput />, div);
  ReactDOM.unmountComponentAtNode(div);
});
