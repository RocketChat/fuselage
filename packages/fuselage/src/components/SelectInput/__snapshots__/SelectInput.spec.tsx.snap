// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[SelectInput Component] renders without crashing 1`] = `
<body>
  <div>
    <label
      class="rcx-box rcx-box--full rcx-label rcx-box rcx-box--full rcx-box--animated rcx-input-box__wrapper"
    >
      <select
        aria-label="select"
        class="rcx-box rcx-box--full rcx-box--animated rcx-input-box--undecorated rcx-input-box--type-select rcx-input-box"
        size="1"
      >
        <option
          class="rcx-box rcx-box--full rcx-input-box__option"
          value="a"
        >
          Item A
        </option>
        <option
          class="rcx-box rcx-box--full rcx-input-box__option"
          value="b"
        >
          Item B
        </option>
        <option
          class="rcx-box rcx-box--full rcx-input-box__option"
          value="c"
        >
          Item C
        </option>
      </select>
      <span
        class="rcx-box rcx-box--full rcx-input-box__addon"
      >
        <i
          aria-hidden="true"
          class="rcx-box rcx-box--full rcx-icon--name-chevron-down rcx-icon rcx-css-4pvxx3"
        >
          ÓÄê
        </i>
      </span>
    </label>
  </div>
</body>
`;
